source/installation_guide/upgrading/from-1.1-to-1.2.rst

```diff
@@ -27,7 +27,7 @@ ACL:
 
 Sieve:
 
- * You should consider `migrating from CMU Sieve to Pigeonhole <https://wiki2.dovecot.org/Pigeonhole/Sieve/Configuration#migration>`_
+ * You should consider :ref:`migrating from CMU Sieve to Pigeonhole <pigeonhole_migration>`
 
```

Comment:
Do we really want to retain v1.x things in our documentation?

Reply:
I don't know, it was in wiki. Do you want to retain this?

Reply:
I don't. Maybe ask around. Depends a bit on the goal of this documentation.